### DISCUSSION TITLE
LiDAR autolanding for a fixed-wing UAV

### DISCUSSION THREAD

**Discussion by Akydaniel7** [community/community#147812] [Programming Help]
### Select Topic Area

Question

### Body

Do you have any idea how to do an auto-landing for a fixed-wing UAV using a 1D Lidar? I mean the code for the pi
I've done it in MissionPlanner, but now I want to add a raspberry pi 4 to the built to process the LiDAR data in order to complete the mission.
I ve got this setup:
Raspberry pi 4
pixhawk 2 black cube
lightware SF000 lidar
Here 3 GPS
here flow lidar & camera
aspd 4525
3S lipo battery
HKPilot Transceiver Telemetry Radio Set V2 (433Mhz)
Frsky X8R receiver
Taranis plus transmitter
1000 kw dc brushless DC motor
30 A ESC
5 sg 90 servomotors

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @Akydaniel7!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
